Story/ArtixWedding.cs

```diff
@@ -0,0 +1,107 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class ArtixWedding
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        ArtixWeddingComplete();
+
+        Core.SetOptions(false);
+    }
+    public void ArtixWeddingComplete()
+    {
+        GrimskullAnnex();
+        ArtixWeddingQuests();
+    }
+
+    public void GrimskullAnnex()
+    {
+        if (Core.isCompletedBefore(3234))
+            return;
+
+        Story.PreLoad();
+
+        //Battle Through the Dungeon 3232
+        Core.Join("ArtixWedding");
+        Story.KillQuest(3232, "GrimskullAnnex", "Grim Fighter");
+
+        //The Way is Blocked 3233
+        Story.KillQuest(3233, "GrimskullAnnex", "Grim Mage");
+
+        //Just a FEW More Monsters... 3234
+        Story.KillQuest(3234, "GrimskullAnnex", "Grim Soldier");
+
+    }
+    public void ArtixWeddingQuests()
```

Comment:
+1 line for prettyness 😆 above the public here or don't il do it later this file is fine either way